### ISSUE TITLE
Icon of 'more actions' menu is not visible in Windows HC mode

### ISSUE THREAD

**Issue by karanbirsingh** [microsoft/accessibility-insights-web#4326]
## Describe the bug

When Windows HC mode is on, the 'more actions' menu is missing its icon:

![screenshot of automated check card, more actions menu icon is blank](https://user-images.githubusercontent.com/7775527/120392244-145ef980-c2e5-11eb-9989-56be821cc36f.PNG)

Expected icon (Windows HC off, AI-Web high contrast theme on)
![screenshot of automated check card, more actions menu icon is visible](https://user-images.githubusercontent.com/7775527/120392188-f98c8500-c2e4-11eb-9f18-891ac995d0db.png)

**To Reproduce**

1. Turn on Windows HC mode (not from within AI-Web)
2. Run automated checks
3. Open the cards view

## Context (please complete the following information)

- OS Name & Version: Windows 10, 19043.985
- AI-Web Version & Environment: 2.26.1, production
- Browser Version: Edge Version 91.0.864.37 (Official build) (64-bit), Chrome 
Version 90.0.4430.212 (Official Build) (32-bit)

- Target Page: any

**Comment by ferBonnin** [comment]
reviewed with @peterdur

**Comment by DaveTryon** [comment]
Confirmed fixed in Canary
